src/ctelnet.h

```diff
@@ -165,6 +168,12 @@ const char NEW_ENVIRON_VAL = 1;
 const char NEW_ENVIRON_ESC = 2;
 const char NEW_ENVIRON_USERVAR = 3;
 
+const char MCCP4_ACCEPT_ENCODING = 1;
+const char MCCP4_BEGIN_ENCODING = 2;
+
+// MCCP4 supported encodings
+const QStringList MCCP4_SUPPORTED_ENCODINGS{"zstd", "deflate"};
```

Comment:
These are non-translatable `QString`s aren't they?:
```suggestion
const QStringList MCCP4_SUPPORTED_ENCODINGS{qsl("zstd"), qsl("deflate")};
```